tools/roxvet/analyzers/declarationcommentmismatch/analyzer.go

```diff
@@ -0,0 +1,104 @@
+package declarationcommentmismatch
```

Comment:
I am not saying it is a bad name, but maybe you had an alternative in mind? 😅

Reply:
I don't know how to express it in a less verbose way that I'm checking whether capitalization between a declaration and a comment mismatches. Any ideas?

Reply:
Maybe just `commentmismatch` I know it's not completely descriptive but I think people will understand it... maybe? 😅

Reply:
Let's call it `commentcapitalization` then

Reply:
Well, let's try to approach this systematically.

(1) The comment before the function will be used of godoc generation (at least for the exported functions). For unexported functions, the comment can be anything (even empty). However, if the comment starts with a word that is the same as function name, we want to make sure that the capitalization matches.

(2) From this description, I am extracting keywords: comment godoc capitalization match.

(3) From the keywords, I try to build something pronounceable: `godoc-comment-capitalization-match`. This could be shortened maybe, so `godoc-capitalization-match`. If the voice must be negative (match vs. mismatch), then maybe `godoc-capitalization-mismatch`. Now just remove the `-` signs, and we have a name.

Feel free to apply this by yourself here. Maybe you will find different keywords?

Reply:
This seems fine, I'll just go with that name then since there's been enough back and forth.